ports/stm/boards/feather_stm32f405_express/pins.c

```diff
@@ -1,5 +1,17 @@
+#include "py/objtuple.h"
 #include "shared-bindings/board/__init__.h"
 
+STATIC const mp_rom_obj_tuple_t sdio_data_tuple = {
```

Comment:
I would prefer that this declaration be brought in line with other board "DEFAULT" devices, defined as a series of macros in the <board>/mpconfigboard.h file, similarly to DEFAULT_I2C_BUS_SCL and the other default busio objects. You can see the implementation of this in `shared-bindings/board/__init__.c`: https://github.com/adafruit/circuitpython/blob/6a46fd5b91d92609cf33eb93f1c4e24632451d25/shared-bindings/board/__init__.c#L51

In this case, you would need to do a pre-processor check to see whether pins beyond DEFAULT_SDIO_D0 exist, and then pass the tuple from there. If you'd like to avoid putting this in the shared-bindings board directory, you could also implement it in port-level common-hal/board files.

Reply:
Could you describe how the user currently declares an SDIO object in python?

Reply:
There is no `board.SDCard()`, which is what these defines are intended to facilitate.

The SDIO SD card is created as:
```
sdioio.SDCard(clock=board.SDIO_CLOCK, command=board.SDIO_COMMAND, data=board.SDIO_DATA)
```

Reply:
board.SDCard() is a honking great idea, but neither stm nor samd have it at the moment.  I entered an issue so that adding it can be prioritized: https://github.com/adafruit/circuitpython/issues/3197